src/components/chart-wrapper/index.js

```diff
@@ -15,10 +15,15 @@ const Wrapper = styled('div')`
   margin: 16px;
 `
 
-const Title = styled('div', forwardRef)`
+const Title = styled('div', forwardRef)((props) =>`
+  font-family: ${props.typography.fontFamily};
+  font-size: ${props.typography.fontSize};
+  font-weight: ${props.typography.fontWeight};
+  color: ${props.typography.color};
+  text-align: ${props.typography.textAlign};
```

Comment:
should we provide reasonable defaults for these typography props?

```js
const Title = styled('div', forwardRef)(({ typography = { ...defaults } }) => `
  font-family: ${typography.fontFamily};
  ...
`)
```

Reply:
I defined default title typography props in shared/constants/title-props.js
`export` const titleDefaultProps = {
  title: '',
  titleStyle: {
    color: 'black',
    fontSize: '18px',
    fontWeight: 'normal',
    textAlign: 'left',
  },
}
```

Reply:
I defined default title typography props in shared/constants/title-props.js
`export const titleDefaultProps = {
  title: '',
  titleStyle: {
    color: 'black',
    fontSize: '18px',
    fontWeight: 'normal',
    textAlign: 'left',
  },
}`